applications/tari_app_grpc/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2018"
 tari_common_types = { version = "^0.8", path = "../../base_layer/common_types"}
 tari_core = {  path = "../../base_layer/core"}
 tari_wallet = {  path = "../../base_layer/wallet"}
-tari_crypto = { git = "ssh://git@github.com/tari-project/tari-crypto.git", branch = "main" }
+tari_crypto = { git = "https://github.com/SWvheerden/tari-crypto.git", rev = "3e8817" } #switch back to official after merge #switch back to official after merge
```

Comment:
Looks like you're using an old commit on your branch here? just use `branch = "sw_comsig"` 

![image](https://user-images.githubusercontent.com/351403/122567085-9753b400-d048-11eb-91cf-74779b542968.png)